### ISSUE TITLE
Update Kotlin and KotlinX.Serialization version

### ISSUE THREAD

**Issue by indrih17** [JakeWharton/retrofit2-kotlinx-serialization-converter#16]
Version 0.4.0 of this library crashed with Kotlin version 1.3.70 because https://github.com/Kotlin/kotlinx.serialization/issues/736
Please, update Kotlin version to 1.3.70 and KotlinX.Serialization version to 0.20.0.

**Comment by nuhkoca** [comment]
@indrih17  you mean the version 0.14.0 by 0.4.0? This plugin had been updated. It shouldn't cause any crash.

**Comment by indrih17** [comment]
@nuhkoca if you have kotlin version 1.3.70 and kotlinx.serialization version 0.20.0, then the converter fails on version 0.4.0.

**Comment by nuhkoca** [comment]
@indrih17 It is not happening for me, interesting!

**Comment by GSala** [comment]
I got this crash. Might be related, but I don't know if it's on this library or on kotlinx.serialization.

```
     Caused by: java.lang.reflect.InvocationTargetException
        at java.lang.reflect.Method.invoke(Native Method)
        at kotlinx.serialization.SerializationKt.invokeSerializerGetter(Serialization.kt:37)
        at kotlinx.serialization.SerializationKt.compiledSerializer(Serialization.kt:11)
        at kotlinx.serialization.PlatformUtilsKt.serializer(PlatformUtils.kt:11)
        at kotlinx.serialization.JvmResolvingKt.serializerByTypeToken(JvmResolving.kt:61)
        at kotlinx.serialization.JvmResolvingKt.serializerByTypeToken(JvmResolving.kt:84)
        at com.jakewharton.retrofit2.converter.kotlinx.serialization.Factory.responseBodyConverter(Factory.kt:23)
        at retrofit2.Retrofit.nextResponseBodyConverter(Retrofit.java:352)
        at retrofit2.Retrofit.responseBodyConverter(Retrofit.java:335)
        at retrofit2.HttpServiceMethod.createResponseConverter(HttpServiceMethod.java:113)
        at retrofit2.HttpServiceMethod.parseAnnotations(HttpServiceMethod.java:82)
        at retrofit2.ServiceMethod.parseAnnotations(ServiceMethod.java:37)
2020-03-05 22:28:51.176 6842-6842/com.saladevs.bytelift E/AndroidRuntime:     at retrofit2.Retrofit.loadServiceMethod(Retrofit.java:192)
        at retrofit2.Retrofit$1.invoke(Retrofit.java:149)
        at java.lang.reflect.Proxy.invoke(Proxy.java:1006)
        	... 16 more
     Caused by: java.lang.NoSuchMethodError: No direct method <init>(Ljava/lang/String;Lkotlinx/serialization/internal/GeneratedSerializer;I)V in class Lkotlinx/serialization/internal/SerialClassDescImpl; or its super classes (declaration of 'kotlinx.serialization.internal.SerialClassDescImpl' appears in /data/app/com.saladevs.bytelift-x1pCW_pzXi5TImI6mGNzlA==/base.apk)
        at ***.Build$$serializer.<clinit>(Build.kt:12)
        at ***.Build$Companion.serializer(Unknown Source:0)
        	... 31 more
```

**Comment by JakeWharton** [comment]
Released 0.5.0 with the fix from @ychescale9.
